templates/events/display.html

```diff
@@ -85,7 +90,13 @@ <h5 class="q-my-none">{{ event_info }}</h5>
 </div>
 
 {% endblock %} {% block scripts %}
+<script src="https://unpkg.com/showdown/dist/showdown.min.js"></script>
```

Comment:
i dont like the fact that we add it via cdn.
maybe we should add it to core via npm aswell, and use it on our `index.html` aswell

Reply:
Check lnbits/lnbits#1956

Reply:
remove it already :)

Reply:
merged